specification.md

```diff
@@ -1631,6 +1631,25 @@ There are two types of conditions for switch states:
 
 These are exclusive, meaning that a switch state can define one or the other condition type, but not both.
 
+At times multiple defined conditions can be evaluated to `true` by runtime implementations.
+This can lead to ambiguity as to which condition should have its transition executed next.
+The specification has two ways of dealing with this ambiguity on the markup level:
+
+  1. order-based: conditions defined first take precedence over conditions defined later.
+  This is backed by the fact that arrays/sequences are orderd in both JSON and YAML.
+  2. priority-based: conditions with higher priority value take precedence over conditions with lower priority.
+  This value is taken from a condition's `priority` parameter.
+
+For example, let's say there are two conditions, A and B, defined in that order. If both A and B are satisfied
```

Comment:
This sentence is a little confusing to me. Could we maybe make it more clear by stating which of the two ways 1. and 2. the example deals with?

For example what I'm thinking:

When using the order-based approach, let's say we have two data / event based conditions A, and B ....
On the other hand when using the priority-based approach and having the same example of two conditions A and B, ........